### PR TITLE
[#223] JSON report

### PR DESCRIPTION
**Pull request by pnatashap**
Formatting report to json format

### CODE REVIEW

**Comment by php-coder** [comment]
nit: consider including into description a special keyword to automatically close related issue (see https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue for details)

File: features/step_definitions/steps.rb [hunk 1]
```diff
@@ -95,6 +95,14 @@
   raise "XML file #{file} doesn't match \"#{xpath}\":\n#{xml}" if xml.xpath(xpath).empty?
 end
 
+Then(/^Text File "([^"]+)" contains "([^"]+)"$/) do |file, substring|
+  raise "File #{file} doesn't exit" unless File.exist?(file)
```
Comment (php-coder) [inline features/step_definitions/steps.rb:99]: nit:
```suggestion
  raise "File #{file} doesn't exist" unless File.exist?(file)
```

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@pnatashap please, tag me in every pull request and issue, if/when you need me to pay attention (like I'm tagging you in this message) -- this way I will get a notification to my inbox and will reply quickly. Many thanks for your contribution!

**Pull request merged**